src/styles/card.scss

```diff
@@ -0,0 +1,144 @@
+@import "./values.scss";
+
+.control {
+  display: grid;
+  place-items: center;
+}
+
+.main {
+  justify-content: center;
+  padding: 50px;
+  width: 944px;
+  height: 862px;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: repeat(2, 1fr);
+  column-gap: 48px;
+  row-gap: 46px;
+  align-items: center;
```

Comment:
width and height should not be overtaken artificially, the indentation should be from the header

```suggestion
  justify-content: center;
  display: grid;
  grid-template-columns: repeat(4, 200px);
  grid-template-rows: repeat(2, 1fr);
  column-gap: 48px;
  row-gap: 46px;
```

u can see it on tests:
![image](https://github.com/mate-academy/layout_catalog/assets/104684446/60e2eef4-58eb-4391-834c-1c9e06367f13)